qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -158,6 +197,18 @@ def _combined_circuits(self, device_layout: bool) -> List[QuantumCircuit]:
                         ) from ex
                     circuit._append(inst, mapped_qargs, mapped_cargs)
 
+                # If scheduling method is alap append shorter sub-circuits with delays
+                if scheduling_method == "asap" and pad_time:
```

Comment:
I think we can remove this because adding delay after measurement, or adding any instruction to qubit without measurement doesn't make any sense. Then name could be something like `delay_sub_circuit_execution: bool`, which can avoid confusion with built-in scheduling logic in preset transpiler.

Reply:
IIRC this is not the case. Backend has meas_map constraints and they align measurements according to the map. I'm not sure they can trigger measurement at different time.